config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :users do
+    resources :posts, only: [:index, :show]
+  end
+  resources :users, only: [:index, :show]
```

Comment:
- [ ] Excellent job setting up controllers and the corresponding views, however, please notice we are exposing unnnecessary endpoints currently for users :point_down: 
![image](https://user-images.githubusercontent.com/97521557/221494956-6de21e9a-e460-450f-ab07-c62bf8f6a055.png)

if we modify the routes like this :point_down: 
```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```
we will expose only the endpoints which are required and functional please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/221495164-cbaeb431-c07a-4358-8647-59b4f56870b0.png)

if we can fix this issue our milestone is done meeting all the requirements :+1: